lib/signs_ui/config/expiration.ex

```diff
@@ -47,22 +56,30 @@ defmodule SignsUi.Config.Expiration do
     {:noreply, state}
   end
 
-  @spec expire_signs(SignsUi.Config.State.t(), (() -> DateTime.t())) :: %{
+  @spec expire_signs(
+          SignsUi.Config.State.t(),
+          (() -> DateTime.t()),
+          (() -> MapSet.t(SignsUi.Alerts.Alert.id()))
+        ) :: %{
           Sign.id() => Sign.t()
         }
-  def expire_signs(state, time_fetcher) do
+  def expire_signs(state, time_fetcher, alert_fetcher) do
     current_dt = time_fetcher.()
+    alert_ids = alert_fetcher.()
```

Comment:
no need to change, but for the future: it's probably easier to test this function if it takes `current_dt` and `alert_ids` rather than functions which return those values.